Cannot find module 'apollo-client' in falcon-ecommerce-uikit

`yarn install` cause:

```
@deity/falcon-ecommerce-uikit: yarn run v1.12.3
@deity/falcon-ecommerce-uikit: $ npm run clean && npm run build:esm && npm run build:cjs && npm run build:tsDeclarations
@deity/falcon-ecommerce-uikit: > @deity/falcon-ecommerce-uikit@0.1.0 clean C:\A\Deity\falcon\packages\falcon-ecommerce-uikit
@deity/falcon-ecommerce-uikit: > rimraf dist
@deity/falcon-ecommerce-uikit: > @deity/falcon-ecommerce-uikit@0.1.0 build:esm C:\A\Deity\falcon\packages\falcon-ecommerce-uikit
@deity/falcon-ecommerce-uikit: > babel src -d dist -x .ts,.tsx -s
@deity/falcon-ecommerce-uikit: Successfully compiled 117 files with Babel.
@deity/falcon-ecommerce-uikit: > @deity/falcon-ecommerce-uikit@0.1.0 build:cjs C:\A\Deity\falcon\packages\falcon-ecommerce-uikit
@deity/falcon-ecommerce-uikit: > rollup -c
@deity/falcon-ecommerce-uikit:
@deity/falcon-ecommerce-uikit: src/index.ts → dist/falcon-ecommerce-uikit.cjs.js...
@deity/falcon-ecommerce-uikit: created dist/falcon-ecommerce-uikit.cjs.js in 2.9s
@deity/falcon-ecommerce-uikit: > @deity/falcon-ecommerce-uikit@0.1.0 build:tsDeclarations C:\A\Deity\falcon\packages\falcon-ecommerce-uikit
@deity/falcon-ecommerce-uikit: > tsc
@deity/falcon-ecommerce-uikit: src/Category/Category.tsx:2:31 - error TS2307: Cannot find module 'apollo-client'.
@deity/falcon-ecommerce-uikit: 2 import { NetworkStatus } from 'apollo-client';
@deity/falcon-ecommerce-uikit:                                 ~~~~~~~~~~~~~~~
@deity/falcon-ecommerce-uikit: src/Checkout/CheckoutLogic.test.tsx:6:30 - error TS2307: Cannot find module 'apollo-client'.
@deity/falcon-ecommerce-uikit: 6 import { ApolloClient } from 'apollo-client';
@deity/falcon-ecommerce-uikit:                                ~~~~~~~~~~~~~~~
@deity/falcon-ecommerce-uikit: src/Query/Query.tsx:4:44 - error TS2307: Cannot find module 'apollo-client'.
@deity/falcon-ecommerce-uikit: 4 import { NetworkStatus, ApolloError } from 'apollo-client';
@deity/falcon-ecommerce-uikit:                                              ~~~~~~~~~~~~~~~
@deity/falcon-ecommerce-uikit: Found 3 errors.
@deity/falcon-ecommerce-uikit: npm ERR! code ELIFECYCLE
@deity/falcon-ecommerce-uikit: npm ERR! errno 2
@deity/falcon-ecommerce-uikit: npm ERR! @deity/falcon-ecommerce-uikit@0.1.0 build:tsDeclarations: `tsc`
@deity/falcon-ecommerce-uikit: npm ERR! Exit status 2
@deity/falcon-ecommerce-uikit: npm ERR!
@deity/falcon-ecommerce-uikit: npm ERR! Failed at the @deity/falcon-ecommerce-uikit@0.1.0 build:tsDeclarations script.
@deity/falcon-ecommerce-uikit: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
@deity/falcon-ecommerce-uikit: npm ERR! A complete log of this run can be found in:
@deity/falcon-ecommerce-uikit: npm ERR!     C:\Users\Marcin\AppData\Roaming\npm-cache\_logs\2019-01-16T10_26_29_834Z-debug.log
@deity/falcon-ecommerce-uikit: error Command failed with exit code 2.
@deity/falcon-ecommerce-uikit: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/examples/shop-with-blog/client/package.json
+++ b/examples/shop-with-blog/client/package.json
@@ -25,7 +25,7 @@
     "apollo-client": "^2.4.2",
     "config": "^2.0.1",
     "formik": "^1.3.1",
-    "graphql": "14.0.2",
+    "graphql": "14.1.0",
     "graphql-tag": "^2.9.2",
     "i18next": "^12.1.0",
     "prop-types": "15.6.2",

--- a/examples/shop-with-blog/client/package.json
+++ b/examples/shop-with-blog/client/package.json
@@ -25,7 +25,7 @@
     "apollo-client": "^2.4.2",
     "config": "^2.0.1",
     "formik": "^1.3.1",
-    "graphql": "^14.0.2",
+    "graphql": "14.0.2",
     "graphql-tag": "^2.9.2",
     "i18next": "^12.1.0",
     "prop-types": "15.6.2",

--- a/packages/falcon-client/package.json
+++ b/packages/falcon-client/package.json
@@ -94,7 +94,7 @@
     "workbox-build": "^3.5.0"
   },
   "devDependencies": {
-    "graphql": "14.0.2",
+    "graphql": "14.1.0",
     "graphql-tag": "^2.9.2",
     "i18next": "^12.1.0",
     "prop-types": "^15.6.2",
@@ -104,7 +104,7 @@
     "react-router-dom": "^4.4.0-beta.6"
   },
   "peerDependencies": {
-    "graphql": "^14.0.2",
+    "graphql": "^14.1.0",
     "graphql-tag": "^2.9.2",
     "i18next": "^12.1.0",
     "prop-types": "^15.6.2",

--- a/packages/falcon-client/package.json
+++ b/packages/falcon-client/package.json
@@ -94,7 +94,7 @@
     "workbox-build": "^3.5.0"
   },
   "devDependencies": {
-    "graphql": "^14.0.2",
+    "graphql": "14.0.2",
     "graphql-tag": "^2.9.2",
     "i18next": "^12.1.0",
     "prop-types": "^15.6.2",

--- a/packages/falcon-ecommerce-uikit/package.json
+++ b/packages/falcon-ecommerce-uikit/package.json
@@ -88,6 +88,7 @@
     "@types/react-router-dom": "4.3.1",
     "@types/react-test-renderer": "^16.0.3",
     "apollo-cache-inmemory": "^1.3.11",
+    "apollo-client": "^2.4.2",
     "apollo-link-schema": "^1.1.2",
     "babel-plugin-annotate-pure-calls": "0.4.0",
     "babel-plugin-graphql-tag": "^1.6.0",

--- a/packages/falcon-magento2-api/package.json
+++ b/packages/falcon-magento2-api/package.json
@@ -39,7 +39,7 @@
     "@deity/falcon-server": "^0.1.0",
     "@deity/falcon-shop-extension": "^0.1.0",
     "eventemitter2": "5.0.1",
-    "graphql": "14.0.2",
+    "graphql": "14.1.0",
     "graphql-tools": "4.0.3",
     "nock": "10.0.4"
   },

--- a/packages/falcon-server-env/package.json
+++ b/packages/falcon-server-env/package.json
@@ -55,7 +55,7 @@
     "apollo-datasource-rest": "0.2.1",
     "apollo-server-caching": "0.2.1",
     "apollo-server-env": "2.2.0",
-    "graphql": "14.0.2",
+    "graphql": "14.1.0",
     "graphql-tools": "4.0.3",
     "html-entities": "1.2.1",
     "lru-cache": "4.1.5",
@@ -63,7 +63,7 @@
     "striptags": "3.1.1"
   },
   "devDependencies": {
-    "@types/graphql": "14.0.3",
+    "@types/graphql": "14.0.5",
     "@types/html-entities": "1.2.16",
     "@types/jest": "23.3.10",
     "@types/koa": "2.0.47",

--- a/packages/falcon-server/package.json
+++ b/packages/falcon-server/package.json
@@ -40,7 +40,7 @@
     "apollo-server-env": "2.2.0",
     "apollo-server-koa": "2.2.6",
     "eventemitter2": "5.0.1",
-    "graphql": "14.0.2",
+    "graphql": "14.1.0",
     "graphql-tools": "4.0.3",
     "koa": "2.6.2",
     "koa-router": "7.4.0",

--- a/packages/falcon-wordpress-api/package.json
+++ b/packages/falcon-wordpress-api/package.json
@@ -15,7 +15,7 @@
   "dependencies": {
     "@deity/falcon-logger": "^0.1.0",
     "@deity/falcon-server-env": "^0.1.0",
-    "graphql": "14.0.2",
+    "graphql": "14.1.0",
     "lodash": "4.17.11",
     "qs": "^6.5.2"
   },

--- a/packages/falcon-wordpress-api/package.json
+++ b/packages/falcon-wordpress-api/package.json
@@ -15,7 +15,7 @@
   "dependencies": {
     "@deity/falcon-logger": "^0.1.0",
     "@deity/falcon-server-env": "^0.1.0",
-    "graphql": "^14.0.2",
+    "graphql": "14.0.2",
     "lodash": "4.17.11",
     "qs": "^6.5.2"
   },

